### ISSUE TITLE
GitLab: MR integration not works with project token

### ISSUE THREAD

**Issue by osipxd** [TeamCodeStream/codestream#661]
If I use [Project Access Token](https://docs.gitlab.com/ee/user/project/settings/project_access_tokens.html) instead of Personal Access Token, I can't open Merge Requests:
```
2021-07-23 09:45:38,176 [1469611]   WARN - agent.CodeStreamLanguageClient - [2021-07-23 06:45:38:176] getMergeRequest — {"providerId":"gitlab/enterprise","pullRequestId":"{\"full\":\"sbi/sbi-android!4\",\"id\":54377}","src":"getPullRequestConversations","force":true}
TypeError: Cannot read property 'mergeRequest' of null
TypeError: Cannot read property 'mergeRequest' of null
    at GitLabEnterpriseProvider.getPullRequest (/snapshot/dist/agent.js:0:0)
    at process._tickCallback (internal/process/next_tick.js:68:7) 
2021-07-23 09:45:38,177 [1469612]   INFO - agent.CodeStreamLanguageClient - [2021-07-23 06:45:38:176] [2da] GitLabEnterpriseProvider.getPullRequest completed • 263 ms 
```

Is it possible to filter only MRs for one project without Project Access Token?
I've not found an option to filter MRs by project [in specification](https://docs.gitlab.com/ee/api/merge_requests.html#list-merge-requests).

**Comment by osipxd** [comment]
I see the ability to filter issues by a project [was added](https://github.com/TeamCodeStream/codestream/pull/648) recently.
But it works for issues only, not for Merge Requests.

**Comment by planteater** [comment]
@osipxd - We'll be extending the project filtering capability to MRs in our next release. I'll update you here as soon as it's available. You will need to use a personal access token, though.

**Comment by planteater** [comment]
This should be addressed in v11.0.13 of CodeStream's extension for JetBrains.
